code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -82,9 +105,28 @@
 			spawn(det_time)
 				prime()
 
+
+/obj/item/weapon/grenade/iedcasing/proc/AddShrapnel(var/obj/item/I, mob/user as mob)
```

Comment:
Snake case. It's a naming standard for vars and procs that produces `stuff_like_this`. What you've used is Pascal case, which does `StuffLikeThis`.